core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3306,8 +3306,9 @@ public String toString() {
       new Builder(Template.WORKER_TIERED_STORE_LEVEL_DIRS_PATH, 0)
           .setDefaultSupplier(() -> OSUtils.isLinux() ? "/mnt/ramdisk" : "/Volumes/ramdisk",
               "/mnt/ramdisk on Linux, /Volumes/ramdisk on OSX")
-          .setDescription("The path of storage directory for the top storage tier. Note "
-              + "for MacOS the value should be `/Volumes/`.")
+          .setDescription("A comma-separated list of paths (eg., /mnt/ramdisk1,/mnt/ramdisk2,/dev/sda1) "
```

Comment:
`/dev/sda1` is a raw block device identifier and is _not_ compatible as a location to store data from the alluxio worker (AFAIK). The path needs to be a device formatted with some filesystem on it. I would suggest just removing this one from the examples or changing it to something else.